tests/strategies/impl/test_fedasync.py

```diff
@@ -0,0 +1,66 @@
+import typing as t
+
+import pytest
+import torch
+
+from flight.strategies import (
+    AggrStrategy,
+    CoordStrategy,
+    DefaultStrategy,
+    NodeState,
+    TrainerStrategy,
+    WorkerStrategy,
+)
+from flight.strategies.base import (
+    DefaultCoordStrategy,
+    DefaultTrainerStrategy,
+    DefaultWorkerStrategy,
+)
+from flight.strategies.impl.fedasync import FedAsync, FedAsyncAggr
+
+
+class TestValidFedAsync:
+    def test_class_hierarchy(self):
+        fedasync = FedAsync(0.5)
+
+        assert (
```

Comment:
Minor note, I would recommend just having several assert statements rather than this using `and` on a bunch of booleans.